src/app.js

```diff
@@ -418,10 +421,18 @@ var run = function () {
     }
   })
 
+  // convert old browser-solidity
```

Comment:
#377 removes a clean way to carry this on, I would say it is not a big deal losing this information